packages/foundry/src/replaceTokens.ts

```diff
@@ -0,0 +1,22 @@
+import path from "node:path";
+import fse from "fs-extra";
+import { glob } from "glob";
+import handlebars from "handlebars";
+
+export async function replaceTokens(files: string[], values: { [key:string]: unknown }, outputDir: string): Promise<void> {
```

Comment:
The nicer way to write `{ [key:string]: unknown }` is `Record<string, unknown>`

Basically la synthaxe d'un dictionnaire c#

https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type

Reply:
Nice! I didn't know this way of writing it.